laravel/resources/views/banners.blade.php

```diff
@@ -38,14 +38,16 @@
 </div>
 @can('add banners')
 <div class="container">
-    <div class="row">
-        <div class="col-lg-3">
-            <button type="button" class="btn btn-primary btn-sm fw-bold" data-bs-toggle="modal" data-bs-target="#addBanner">
-                Add Banner
-            </button>
+    @if($instance_list->count() != 0)
```

Comment:
Makes more sense, doesn't it? 😅 
```suggestion
    @if($instance_list->count() > 0)
```